# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1322013366 28800
#      Tue Nov 22 17:56:06 2011 -0800
# Node ID 4b1ce071f5046ccd476dbcc86859a5a3106e1968
# Parent  521adfd775be19d32bc4afa8c7fd5bd70bbabaa5
test.m: Update tests which depend on error messages changed in toeplitz.m

* test.m: Update tests which depend on error messages changed in toeplitz.m

diff --git a/scripts/testfun/test.m b/scripts/testfun/test.m
--- a/scripts/testfun/test.m
+++ b/scripts/testfun/test.m
@@ -672,23 +672,24 @@ endfunction
 % !testif HAVE_FOOBAR
 % ! ## missing feature. Fail if this test is run
 % ! error("Failed missing feature test");
 
 ### Test for a known failure
 %!xtest error("This test is known to fail")
 
 ### example from toeplitz
-%!shared msg
-%! msg="expecting vector arguments";
-%!fail ('toeplitz([])', msg);
-%!fail ('toeplitz([1,2],[])', msg);
-%!fail ('toeplitz([1,2;3,4])', msg);
-%!fail ('toeplitz([1,2],[1,2;3,4])', msg);
-%!fail ('toeplitz ([1,2;3,4],[1,2])', msg);
+%!shared msg1,msg2
+%! msg1="C must be a vector";
+%! msg2="C and R must be vectors";
+%!fail ('toeplitz([])', msg1);
+%!fail ('toeplitz([1,2;3,4])', msg1);
+%!fail ('toeplitz([1,2],[])', msg2);
+%!fail ('toeplitz([1,2],[1,2;3,4])', msg2);
+%!fail ('toeplitz ([1,2;3,4],[1,2])', msg2);
 % !fail ('toeplitz','usage: toeplitz'); # usage doesn't generate an error
 % !fail ('toeplitz(1, 2, 3)', 'usage: toeplitz');
 %!test  assert (toeplitz ([1,2,3], [1,4]), [1,4; 2,1; 3,2]);
 %!demo  toeplitz ([1,2,3,4],[1,5,6])
 
 ### example from kron
 %!#error kron  # FIXME suppress these until we can handle output
 %!#error kron(1,2,3)
